BatchApex/BatchLeadSchedule.cls

```diff
@@ -0,0 +1,9 @@
+/**
+ * Schedulable class for to delete the duplicate leads
+ */
+global with sharing class BatchLeadSchedule implements Schedulable {
```

Comment:
From where are you scheduling this? this needs to be scheduled after every 3 hours, Where is the cron string? and from where are you scheduling this?